src/CMakeLists.txt

```diff
@@ -223,6 +223,22 @@ if(MSVC)
     set_target_properties(woof-setup PROPERTIES LINK_FLAGS_DEBUG "/SUBSYSTEM:CONSOLE")
 endif()
 
+if(WIN32)
+    add_executable(conredir WIN32 "../win32/win_console_wrapper.c")
+    target_woof_settings(conredir)
+    if(MSVC)
+        set_target_properties(conredir PROPERTIES
+                              LINK_FLAGS "/MANIFEST:NO /SUBSYSTEM:CONSOLE")
+    else()
+        set_target_properties(conredir PROPERTIES LINK_FLAGS "-municode -mconsole")
+    endif()
+
+    add_custom_command(TARGET conredir POST_BUILD
+            COMMAND "${CMAKE_COMMAND}" -E copy
+            "$<TARGET_FILE_DIR:woof>/conredir.exe"
+            "$<TARGET_FILE_DIR:woof>/woof.com" VERBATIM)
```

Comment:
Can you simply rename a .com file to .exe?

Reply:
Or vice versa?

Reply:
Do you mean renaming the target to `woof.com`? I don't think it will work.

Reply:
They are incompatible in DOS. This is just `cmd.exe` hack (the Windows console interpreter). It is backwards compatible with DOS since Win9x and looks for the `.com` file first.

Reply:
So, this hack only works if you start the game by calling it as "woof", but it won't work if you explicitly call it as "woof.exe"?

Reply:
Yes, right.